packages/sn-document-viewer-react/src/hooks/usePreviewImage.ts

```diff
@@ -0,0 +1,18 @@
+import { useCallback } from 'react'
+import { usePreviewImages } from './usePreviewImages'
+
+export const usePreviewImage = (pageNo: number) => {
+  const images = usePreviewImages()
+  const image = images.imageData.find(i => i.Index === pageNo)
+  const { imageData, ...context } = { ...images }
+
+  // ToDo: Polling?
```

Comment:
do we need to talk about this?

Reply:
@gallayl can you please help us solve this riddle? 😄

Reply:
🃏 I think the issue was that if you have only a few preview images generated (e.g. the first 10 only) but you want to see the 11th, you have to start a similar polling as for document initialization but with the exact page Id. I'm not sure that this logic was moved from Redux to another hook but I think it should be here :)

Reply:
Thanks, I've fixed that.